### DISCUSSION TITLE
It worked for few hours and just like magic: Copilot is encountering temporary issues

### DISCUSSION THREAD

**Discussion by baruchsheffi** [community/community#10819] [Copilot Conversations]
I tried everything written here, re-installing, recreated credentials, I even installed UBUNTU desktop on a virtual machine (aka new and cleaned OS) installed VS Code, disabled the sync just to make sure, installed Github CLI > logged-in, installed Copilot on VS code
 allow... agree.... tested... and got the same error :(
![Screenshot 2022-01-31 182351](https://user-images.githubusercontent.com/797544/151832228-300cbcf6-fbf7-44a8-ae6a-2b501feb72f6.jpg)

when it was working, it was great... now I feel like someone took my friend :)

ow... and disabled the firewall as well...

**Comment by davecheney** [comment]
Please ignore the temporary error warning, the code that triggers that is overly sensitive.

**Reply by baruchsheffi** [reply]
But it stopped working and doesn't give any suggestions...

**Reply by baruchsheffi** [reply]
furthermore, checkout the screenshot I put on the top, I did as the getting started doc says: wrote function calculateDaysBetweenDates(begin, end)
and... nothing happens but the warning  and "Breaking, no choices"

**Reply by davecheney** [reply]
as you’ve discovered, the “copilot is having temporary issues” message is related to not having any choices. This is a bug, we thought it had been fixed, but we need to fix it some more as that warning is spurious. 

Copilot is working, it just has nothing to offer at the moment. Type more, add a comment, give it some more context and it may have something to suggest.

**Reply by baruchsheffi** [reply]
https://user-images.githubusercontent.com/797544/151962258-bbaf64d3-df65-4bf8-a281-77204697bc26.mp4

+ want me to share screen with you?

**Reply by davecheney** [reply]
Thank you for providing this capture. I'm going to escalate this internally and see if we can figure out what's going on.

**Reply by baruchsheffi** [reply]
Great! thanks! if you need my assistance in anyway, let me know, ill provide anydesk access if needed, just get my new friend back to work :)

**Reply by davecheney** [reply]
Can you do be a favor and signout and back in to vscode 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by baruchsheffi** [reply]
same...

https://user-images.githubusercontent.com/797544/152119307-8a882949-c610-43a8-8e87-4f08ec49e399.mp4

what you cannot see in the movie:
1. After I clicked the sign-in it opened the browser, I allowed it to open back in code
2. After stopping the recording, I signed in to github pull requests and issues - it didn't change anything on copilot :)

Again, if you'd like accessing in Anydesk or similar to investigate my side, please let me know, I have no problem with that as long as you bring my buddy back to work :)

**Reply by davecheney** [reply]
Thank you for the additional details. We’re still investigating what’s happened; basically your client never gets a token to talk the copilot and so the rest of the conversation, complaints about temporary errors and no completions is garbage. It doesn’t seem to be a widespread problem, but I don’t have anything more to add this evening.

**Reply by baruchsheffi** [reply]
It's noon here :)
I'll try on a different PC, to be sure, ill try my MacBook and keep you posted... after that I'll escalate to TCP port listener...
Is there any verbose log stored anywhere on the local drive?

**Reply by baruchsheffi** [reply]
Hi again, i tested it on my MacBook, same result.. i even disabled the sync settings before installing any extension.
Should i create a new user on Github?
![copilot](https://user-images.githubusercontent.com/797544/152224296-9f5911c1-6106-4e31-a86b-6ec5bcc0ebf7.jpg)

**Reply by davecheney** [reply]
I've found the problem. The fix has been deployed. It should be working for you again. Thanks for your patience.

**Reply by baruchsheffi** [reply]
Ok, no I'm getting: [ERROR] [default] [2022-02-03T08:06:58.500Z] GitHub Copilot could not connect to server. Extension activation failed: "Unexpected end of JSON input"

I tried reinstalling, tried sign-out sign-in, tried recreate the guthub credential on Windows... everything ends up with same result:
GitHub Copilot could not connect to server. Extension activation failed: "Unexpected end of JSON input"

P.S
I get the error before I get the message to agree/allow the terms... after login from the browser it gives the message immediately 

P.P.S
I just tried on my MacBook, again, same result... "Unexpected end of JSON input"

**Comment by sidm1983** [comment]
@davecheney I am having the same issue. Every time I type code, I can see the little copilot icon in the status bar change to a spinning loader and then when it is done, I still get no suggestions. I was following the first suggestion code from the copilot getting started readme (i.e. `function getDaysBetweenDates(begin, end) {`), but I didn't get a suggestion.

Here is my Github Copilot output:
```
[INFO] [default] [2022-02-03T07:46:57.792Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest
[INFO] [default] [2022-02-03T07:46:58.021Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest/completions] took 228 ms
[INFO] [default] [2022-02-03T07:46:58.021Z] prompt: {"prefix":"// Language: javascript\nfunction getDaysBetweenDates(begin, end) {}","suffix":""}
[INFO] [streamChoices] [2022-02-03T07:46:58.022Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-4XR7Reqaau2gR5Z6gZ1Okmp1O1JZ3}] created: [{1643874417}]
[INFO] [ghostText] [2022-02-03T07:46:58.022Z] Breaking, no choices
```

I then tried adding a comment and then it started giving me suggestions, but line by line suggestions that I had to accept one at a time before it would give me the next one. Is it meant to work like that? Any insights would be greatly appreciated. Will continue to test and report back any other findings.

Thank you!

**Reply by davecheney** [reply]
@sidm1983 would you be so good as to start a new discussion. I believe your issue is not the same as the OP’s

**Reply by sidm1983** [reply]
@davecheney done! Here is the new discussion: #10948
